ONSdigital/ras-frontstage
# Help journey sign in

# What and why?
This PR updates the sign in page as part of the update to the survey help journey. 
Some of the spacing is a little strange in the figma with overlapping text fields on error messages so i have stuck with design system components which means the spacing doesn't match exactly to the design docs.  

# How to test?
Run the tests
Check the updated page against the figma designs and ensure everything works and looks appropriate given the design. 
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-1293
